akka-docs/rst/scala/stream/stages-overview.rst

```diff
@@ -587,6 +587,17 @@ when the predicate first return false and then complete.
 
 **completes** when predicate returned false or upstream completes
 
+takeUntil
```

Comment:
Not very efficient, but it's possible to express this with mapConcat and ordinary takeWhile